File: msgraph-mail/typescript/src/users/mailFolders/sensitivity.ts
import {SerializationWriter, ParseNode, Parsable} from '@microsoft/kiota-abstractions';

export class Sensitivity implements Parsable<Sensitivity> {
    public serialize (writer: SerializationWriter) : void {
    };
    public deserializeFields () : Map<string, (item: Sensitivity, node: ParseNode) => void> {
        return new Map<string, (item: Sensitivity, node: ParseNode) => void>([
        ]);
    };
}

File: msgraph-mail/dotnet/Users/MailFolders/Sensitivity.cs
using Kiota.Abstractions.Serialization;
using System;
using System.Collections.Generic;
using System.Linq;
namespace Graphdotnetv4.Users.MailFolders {
    public class Sensitivity : IParsable<Sensitivity> {
        public IDictionary<string, Action<Sensitivity, IParseNode>> DeserializeFields => new Dictionary<string, Action<Sensitivity, IParseNode>> {
        };
        public void Serialize(ISerializationWriter writer) {
        }
    }
}
